frontend/src/metabase/home/components/HomeContent/HomeContent.unit.spec.tsx

```diff
@@ -68,8 +68,11 @@ const setup = async ({
 
 describe("HomeContent", () => {
   beforeEach(() => {
-    jest.useFakeTimers({ advanceTimers: true });
-    jest.setSystemTime(new Date(2020, 0, 10));
+    jest.useFakeTimers({
+      advanceTimers: true,
+      now: new Date(2020, 0, 10),
+      doNotFake: ["setTimeout"],
```

Comment:
not faking setTimeout still allows us to fake the current date. Another way to handle this may be to use a different time for `first_login` when creating the mock user